### PR TITLE
docs: document release and bugfixing process

### PR DESCRIPTION
**Pull request by pchila**
Signed-off-by: Paolo Chila <paolo.chila@dynatrace.com>

### CODE REVIEW

File: RELEASE.md [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+new directory on disk, increase the major version both in the directory name
+and `task.json` file and then implement the breaking change in the new version.
+
+## How to release
+
+### Typical case: create a new version based on latest release
+
+New versions are created using `Pre-Release` and `Release` workflows available
+in the [`Actions`
+tab](https://github.com/keptn-sandbox/keptn-jenkins-library/actions) on GitHub.
```
Comment (christian-kreuzberger-dtx) [inline RELEASE.md:94]: Wrong link :)

File: RELEASE.md [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+The worflows will take care of calculating the next version number and create a
+GitHub release depending on the type of changes introduced.
+
+Workflows must be run on the appropriate branch (`master` should be the normal
```
Comment (christian-kreuzberger-dtx) [inline RELEASE.md:98]: Let's state this as `master`/`main` (in case we switch to `main` as the default branch)

**Review by christian-kreuzberger-dtx — DISMISSED** [review]
Left two comments, other then that I believe this is representing the workflow and steps needed by VS Marketplace

File: RELEASE.md (hunk 1 above)
Comment (pchila) [reply RELEASE.md:94]: Oups... quick copy/past error :sweat_smile:

**Review by christian-kreuzberger-dtx — APPROVED** [review]
LGTM

**Pull request merged**